README.md

```diff
@@ -37,4 +38,4 @@ node {
 
 ## Jenkins core compatibility
 
-This plugin requires Jenkins 1.609+ (since 1.5 version of plugin).
+This plugin requires Jenkins 2.164.1+ (since 1.8 version of plugin).
```

Comment:
The next release will be 1.7. can you just point to 1.7 or just remove the information in parentheses?